pkg/skyenv/values.go

```diff
@@ -59,8 +59,9 @@ const (
 const (
 	DmsgPtyPort uint16 = 22
 
-	DefaultDmsgPtyCLINet  = "unix"
-	DefaultDmsgPtyCLIAddr = "/tmp/dmsgpty.sock"
+	DefaultDmsgPtyCLINet    = "unix"
+	DefaultDmsgPtyCLIAddr   = "/tmp/dmsgpty.sock"
+	DefaultDmsgPtyWhitelist = DefaultSkywirePath + "/dmsgpty/whitelist.json"
```

Comment:
I think we recently got rid of the separate whitelist. Didnt we @ersonp ?

Reply:
> @ersonp @0pcom did any of you test this on an actual board? Overall good PR. Thanks.
> 
> @0pcom Please document the usage with `-r` flag and do not automatically set it.

Will do

Reply:
yes we got rid of the whitelist.